aldryn_config.py

```diff
@@ -0,0 +1,11 @@
+from aldryn_client import forms
+
+
+class Form(forms.BaseForm):
+    def to_settings(self, data, settings):
+        settings['INSTALLED_APPS'].extend([
+            'djangocms_content_expiry',
```

Comment:
```suggestion
```

I think that this entry will cause installation issues, none of the other addons add themselves, I suspect the internals of Aldryn / Divio will add the underscored entry for the current package. Aldryn-django does this because it would resolve to: aldryn_django and not django itself.